core/modules/date/tests/date_field.test

```diff
@@ -417,17 +422,23 @@ class DateFieldTokenTestCase extends DateFieldBasic {
     foreach ($tokens as $name => $expected) {
       $token = $input[$name];
       if (!isset($expected)) {
-        $this->assertTrue(!isset($values[$token]), t("Token value for @token was not generated.", array('@token' => $token)));
+        $this->assertTrue(!isset($values[$token]), t("Token value for @token was not generated.", array(
+          '@token' => $token,
+        )));
       } else {
```

Comment:
Sorry I didn't pick this earlier:

```suggestion
      }
      else {
```

Reply:
OK, thanks,  I'll change this... but this is code I haven't touched. If we start modifying things we haven't touched or have not been pointed out by the coding standard test, the PRs become long, bloated, and hard to review.